geoapps/inversion/electricals/direct_current/pseudo_three_dimensions/constants.py

```diff
@@ -0,0 +1,338 @@
+    "resolution": None,
+    "max_chunk_size": 128,
+    "chunk_by_rows": True,
+    "mesh": None,
+    "u_cell_size": 25.0,
+    "v_cell_size": 25.0,
+    "depth_core": 500.0,
+    "horizontal_padding": 1000.0,
+    "vertical_padding": 1000.0,
+    "expansion_factor": 1.1,
+    "window_center_x": None,
+    "window_center_y": None,
+    "window_width": None,
+    "window_height": None,
+    "window_azimuth": None,
+    "parallelized": True,
+    "n_cpu": None,
+    "out_group": "DirectCurrentForward",
+    "monitoring_directory": None,
+    "workspace_geoh5": None,
+    "run_command": "geoapps.inversion.driver",
+    "run_command_boolean": False,
+    "conda_environment": "geoapps",
+    "distributed_workers": None,
+    "gradient_type": "total",
+    "alpha_s": 1.0,
+    "alpha_x": 1.0,
+    "alpha_y": 1.0,
+    "alpha_z": 1.0,
+    "s_norm": 0.0,
+    "x_norm": 2.0,
+    "y_norm": 2.0,
+    "z_norm": 2.0,
+}
+
+inversion_ui_json = {
+    "potential_channel_bool": True,
+}
+
+forward_ui_json = {
+    "gradient_type": "total",
+    "alpha_s": 1.0,
+    "alpha_x": 1.0,
+    "alpha_y": 1.0,
+    "alpha_z": 1.0,
+    "s_norm": 0.0,
+    "x_norm": 2.0,
+    "y_norm": 2.0,
+    "z_norm": 2.0,
+}
+
+default_ui_json = {
+    "title": "SimPEG Direct Current inversion",
+    "inversion_type": "direct current 2d",
+    "line_object": {
+        "association": ["Cell", "Vertex"],
+        "dataType": "Referenced",
+        "group": "Data",
+        "main": True,
+        "label": "Line field",
+        "parent": "data_object",
+        "value": None,
+    },
+    "line_id": 1,
+    "data_object": {
+        "main": True,
+        "group": "Data",
+        "label": "Object",
+        "meshType": "{275ecee9-9c24-4378-bf94-65f3c5fbe163}",
+        "value": None,
+    },
+    "potential_channel_bool": True,
+    "potential_channel": {
+        "association": ["Cell", "Vertex"],
+        "dataType": "Float",
+        "group": "Data",
+        "main": True,
+        "label": "Potential channel",
+        "parent": "data_object",
+        "value": None,
+    },
+    "potential_uncertainty": {
+        "association": ["Cell", "Vertex"],
+        "dataType": "Float",
+        "group": "Data",
+        "main": True,
+        "isValue": True,
+        "label": "Potential uncertainty",
+        "parent": "data_object",
+        "property": None,
+        "value": 1.0,
+    },
+    "starting_model_object": {
+        "group": "Starting Model",
+        "main": True,
+        "meshType": "{C94968EA-CF7D-11EB-B8BC-0242AC130003}",
+        "optional": True,
+        "enabled": False,
+        "label": "Object",
+        "value": None,
+    },
+    "starting_model": {
+        "association": "Cell",
+        "dataType": "Float",
+        "group": "Starting Model",
+        "main": True,
+        "isValue": False,
+        "parent": "starting_model_object",
+        "label": "Conductivity (Siemens/m)",
+        "property": None,
+        "value": 0.0,
```

Comment:
Hey would you mind changing this here and for the 3D case, zero is bad for conductivity

Reply:
the real value (1e-1) from the defaults dictionary overrides this values anyway, but I've replaced everywhere in the dc packages just for show.